### ISSUE TITLE
main.cpp compilation

### ISSUE THREAD

**Issue by hgeorge21** [alecjacobson/computer-graphics-kinematics#24]
All the `v.core` should be `v.core()`

**Comment by abhimadan** [comment]
Yes, this has been fixed upstream already. Please pull to get the changes.

**Comment by abhimadan** [comment]
Oh wait this is the a7 repo...we only fixed this for a4. Thanks for letting me know, I'll go through the remaining assignment repos and fix any lurking compile errors.

**Comment by abhimadan** [comment]
On second thought, please hold off on starting future assignments (like this one) for now. We're slightly changing them from last year, so we'll go through them and apply fixes as appropriate before they're released, so that students will get a working version when they start the assignment. In case you clicked this repo by mistake, the current assignment repo is [here](https://github.com/alecjacobson/computer-graphics-bounding-volume-hierarchy).
